ktor-io/js/src/io/ktor/utils/io/js/TextDecoder.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.utils.io.js
+
+import org.khronos.webgl.*
+import kotlin.reflect.*
```

Comment:
Why do you need `reflect`?

Reply:
Thanks, will check

Reply:
Removed. IDE didn't show it as unused for some reason